pennylane/gradients/pulse_generator_gradient.py

```diff
@@ -407,16 +407,15 @@ def _pulse_generator(tape, argnum=None, shots=None, atol=1e-7):
     This method combines automatic differentiation of few-qubit operations with
     hardware-compatible shift rules.
     It allows for the evaluation of parameter-shift gradients for many-qubit pulse programs
-    on hardware, with the limitation that the program must be composed of few-qubit pulses.
+    on hardware, with the limitation that the individual pulses must be acting on a reduced number of qubits.
```

Comment:
Maybe a bit more "concrete"?
```suggestion
    on hardware, with the limitation that the individual pulses must be acting on a 
    sufficiently small number of qubits.
```